headers/components/temp_doeclim_component.hpp

```diff
@@ -81,40 +81,30 @@ class TempDOECLIMComponent : public IModelComponent {
     double flnd;             // fractional land area
     double fso;              // ocean fractional area below 60m
     
-    //Intermediate DOECLIM parameters
+    // Intermediate DOECLIM parameters
```

Comment:
What does "Intermediate" mean here?

Reply:
These are the DOECLIM parameters that are calculated from the above constants and are in turn used to calculate the model outputs. Perhaps there's a better way to label them?

Reply:
"DOE parameters calculated from constants above" ?